packages/workflows/src/helper/workflow-export.ts

```diff
@@ -83,7 +82,7 @@ export const exportWorkflow = <TData = unknown, TResult = unknown>(
         context
       )
 
-      const errors = transaction.getErrors(TransactionHandlerType.INVOKE)
```

Comment:
revert this hehe

Reply:
ah damn, I thought I had returned the compensation errors in a different array

Reply:
done